engine/src/org/pentaho/di/trans/steps/gettablenames/GetTableNamesMeta.java

```diff
@@ -251,6 +282,11 @@ public boolean isAddSchemaInOut() {
     return this.addSchemaInOutput;
   }
 
+  @Injection( name = "CONNECTIONNAME" )
+  public void setConnection( String connectionName ) {
+	  database = DatabaseMeta.findDatabase( databases, connectionName );
```

Comment:
@jmwoliver There is a checkstyle violation on this line.  I'll show you how to fix this tomorrow.